### ISSUE TITLE
Interpreting linear models R^2

### ISSUE THREAD

**Issue by janxue** [STAT540-UBC/Discussion#311]
Hi there,

I have tried to fit the values for principle component 1 based on batch however, I am confused about which R^2 value to use?

R^2 measures how good the fit is to the data. However, how should I think about the adjusted R^2 value vs. the multiple R^2 value. 

I think that the adjusted R^2 value takes into account the number of predictors, but in this case I only have one predictor (batch) so maybe it doesn’t matter as much?

Below is the summary of the linear model. 

-------------
Call:
lm(formula = PC1 ~ batch, data = prinComp110)

Residuals:
     Min         1Q           Median        3Q       Max 
-0.33683 -0.16755 -0.07214  0.19900  0.33122 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)
(Intercept)  0.06804    0.08448   0.805    0.432
batch       -0.07655    0.07029  -1.089    0.292

Residual standard error: 0.2412 on 16 degrees of freedom
Multiple R-squared:  0.06901,	Adjusted R-squared:  0.01082 
F-statistic: 1.186 on 1 and 16 DF,  p-value: 0.2923

**Comment by jafarzadeh91** [comment]
H Jan, 
you are right. adjusted R-square takes into accounts the number of the predictors in calculating the R-square because a greater number of predictors always lead to a better R-square by increasing the degree of freedom in the model.
Using adjusted R-square makes it easy to compare models with different numbers of predictors.
In your case, you have **two** predictors: batch and Intercept, so I guess the difference is because of that.

**Comment by wilcas** [comment]
Hi Jan,

The Wikipedia page for the [coefficient of determination](https://en.wikipedia.org/wiki/Coefficient_of_determination) actually does a pretty good job of explaining this. 

TL;DR: Multiple R-Squared is telling you how much of the variation in PC1 is explained by variation in batch, where as the adjusted R-squared does roughly the same thing but accounts for how R-squared tends to improve with more predictors. 

If you want to discuss the fraction of variance being explained in your response (y, or PC1 in this case) by your predictors (batch in this case), using Multiple R-squared is a good choice. However, if you want to compare models to one another that have different numbers of predictors, adjusted R-squared will be a better choice. In adjusted R-squared we account for the number of samples we have, as well as the number of predictors we are using to predict our response.

**Comment by janxue** [comment]
Hi Will and Sina,

This makes sense, thank you!


Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
